### DISCUSSION TITLE
Project Feedback: Multiple Groups

### DISCUSSION THREAD

**Discussion by fishcharlie** [community/community#8756] [Projects and Issues]
It would be great if there was a way to group by both status, and repository (https://github.com/github/roadmap/issues/283). Where status is the top level group, and repository is a secondary group.

This would allow me to easily see by repository which items are in Todo. Additionally, it would be useful to be able to rearrange/sort the repository groups by dragging the order of them. This would allow me to prioritize based on repository, and then items within the repository.

**Comment by stevemar** [comment]
+1. This would be a great feature. It would be really useful to be able to apply multiple "group by" fields to a single view.

**Comment by Jay-o-Way** [comment]
Just quickly read this, but - wouldn't a new View (or tab) be also a solution?
